_data/versions.yml

```diff
@@ -1,3 +1,30 @@
+- title: v3.16.5
+  tigera-operator:
+   image: tigera/operator
+   registry: quay.io
+   version: v1.10.8
```

Comment:
This will exist after https://github.com/tigera/operator/pull/990 is merged and a 1.10.8 is released